Fixes #10091 - Document WDAC audit mode experimental feature (#10094)
MicrosoftDocs/PowerShell-Docs: reference/docs-conceptual/learn/security-features.md

PowerShell 7.3 extends the data that's sent to AMSI for inspection. It now includes all invocations
of .NET method members.

For more information about AMSI, see [How AMSI helps][11].

## Constrained language mode

**ConstrainedLanguage** mode protects your system by limiting the cmdlets and .NET types that can be
used in a PowerShell session. For a full description, see [about_Language_Modes][04].

## Application Control

Windows 10 includes two technologies, [Windows Defender Application Control (WDAC)][10] and

> [AppLocker][07] that can be used for application control. They allow you to create a lockdown

Reviewer: ```suggestion
[AppLocker][07] that you can use to control applications. They allow you to create a lockdown
```
